### PR TITLE
centos6 nginx

### PR DESCRIPTION
**Pull request by jburel**
Add nginx section following re-organisation

The walkthroughs in omero-install have been re-organized to allow a split of the doc between server and web
Some dependencies/variables were no longer installed/set for centos 6 as a consequence.

Since we won't have a installation of omeroweb separately from omeroserver for centos6, i have kept the actual structure

### CODE REVIEW

**Comment by rleigh-codelibre** [comment]
This all reads fine and the rendered page looks good.  It looks fine to merge as is.

One thing which I think would be worthwhile to add, however,  is a clear note at the top of the page after the first paragraph, to state that "CentOS 6 is deprecated and will be dropped for the next major release.  CentOS 7 is preferable for new installations."  Or words to that effect.

File: omero/sysadmins/unix/server-centos6-ice36.rst [hunk 1]
```diff
@@ -5,7 +5,9 @@ This is an example walkthrough for installing OMERO on CentOS 6 with Python 2.7,
 a dedicated system user, and should be read in conjunction with
 :doc:`server-installation` and :doc:`install-web`. You can use this as a guide
 for setting up your own test server. For production use you should also read
-the pages listed under :ref:`index-optimizing-server`.
+the pages listed under :ref:`index-optimizing-server`. However CentOS 6 is deprecated,
```
Comment (rleigh-codelibre) [inline omero/sysadmins/unix/server-centos6-ice36.rst]: Might be worth using a `.. note` or `.. warning` to call this out.

File: omero/sysadmins/unix/server-centos6-ice36.rst [hunk 2]
```diff
@@ -23,6 +22,10 @@ This guide describes how to install the **recommended** versions, not all
 the supported versions.
 This should be read in conjunction with :doc:`../version-requirements`.
 
+.. warning::
+
+   CentOS 6 is deprecated, CentOS 7 is preferable for new installations
```
Comment (rleigh-codelibre) [inline omero/sysadmins/unix/server-centos6-ice36.rst]: "See" should be a new sentence or separated with a semi-colon from the preceding text.

**Comment by jburel** [comment]
@rleigh-codelibre done

**Comment by rleigh-codelibre** [comment]
Thanks, this looks fine.  Please merge.

**Pull request merged**